airflow/models/dagrun.py

```diff
@@ -97,12 +94,11 @@ def set_state(self, state):
 
     @declared_attr
     def state(self):
-        return synonym('_state',
-                       descriptor=property(self.get_state, self.set_state))
+        return synonym('_state', descriptor=property(self.get_state, self.set_state))
 
     @classmethod
-    def id_for_date(cls, date, prefix=ID_FORMAT_PREFIX):
-        return prefix.format(date.isoformat()[:19])
+    def id_for_date(cls, date, prefix=DagRunType.SCHEDULED.value):
```

Comment:
Why do we need this method?

Reply:
Hm, it seems to be used only in tests. Should I remove it?

Reply:
Yes. Yes. Fstrings are a better idea. It's clearer

Reply:
![image](https://media3.giphy.com/media/b7f0X8Okk1uyk/source.gif)